src/content/posts/post-2.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Why a balloon is a flexible bag
```

Comment:
What about this post? Delete all posts that are already here, other than the one you created